mandatory/3-function-output.js

```diff
@@ -3,12 +3,17 @@ function getRandomNumber() {
   return Math.random() * 10;
 }
 
+// Math.random() can be used to return random integers.
+
 // Add comments to explain what this function does. You're meant to use Google!
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
+// The concat() method concatenates the string arguments to the calling string and returns a new string.
+
 function concatenate(firstWord, secondWord, thirdWord) {
+  return  firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
It's great that you've clearly grasped concat, but do you actually need it here? Could you return the strings without it?

Reply:
I guess I can concatenate them using +